app/views/book/show.html.erb

```diff
@@ -16,11 +16,13 @@
     <%= @book.description_html %>
   </div>
 
-  <div class="actions">
-    <a href="<%= lesson_path(@book.first_chapter.first_lesson) %>" class="btn btn-success">
-      <%= t(:start_practicing) %>
-    </a>
-  </div>
+  <% if @next_exercise.present? %>
+    <div class="actions">
```

Comment:
@flbulgarelli I know that, but I'd rather use `present?` method, it is more expressive to me. I won't modify this, if you disagree please let me know!